### PR TITLE
Corrected a few typos in the documentation and comments

### PR DESCRIPTION
**Pull request by tgiphil**

### CODE REVIEW

File: src/jit/sideeffects.cpp [hunk 1]
```diff
@@ -337,7 +337,7 @@ bool AliasSet::InterferesWith(const AliasSet& other) const
 //    Returns true if the reads and writes in this alias set interfere
 //    with those for the given node.
 //
-//    An alias set interferes with a given node iff it interferes with the
```
Comment (AaronRobinsonMSFT) [inline src/jit/sideeffects.cpp]: ["iff"](https://en.wikipedia.org/wiki/If_and_only_if) isn't a typo - it means "if and only if".

File: src/jit/sideeffects.cpp [hunk 2]
```diff
@@ -529,7 +529,7 @@ bool SideEffectSet::InterferesWith(const SideEffectSet& other, bool strict) cons
 //    Returns true if the side effects in this set interfere with the side
 //    effects for the given node.
 //
-//    A side effect set interferes with a given node iff it interferes
```
Comment (AaronRobinsonMSFT) [inline src/jit/sideeffects.cpp]: Not a typo

File: src/jit/sideeffects.cpp (hunk 1 above)
Comment (tgiphil) [reply src/jit/sideeffects.cpp]: Corrected. Thanks.

**Comment by AaronRobinsonMSFT** [comment]
@dotnet-bot test Windows_NT x64 Formatting
@dotnet-bot test Ubuntu x64 Formatting

File: src/jit/lower.cpp [hunk 3]
```diff
@@ -3661,7 +3661,7 @@ void Lowering::InsertPInvokeCallProlog(GenTreeCall* call)
 
     InsertTreeBeforeAndContainCheck(insertBefore, storeLab);
 
-    // Push the PInvoke frame if necessary. On 32-bit targets this only happens in the method prolog if a method
+    // Push the PInvoke frame if necessary. On 32-bit targets this only happens in the method prologue if a method
```
Comment (BruceForstall) [inline src/jit/lower.cpp]: We use "prolog" without the trailing "ue" everywhere. Please don't change this.

**Comment by BruceForstall** [comment]
Note the "formatting" jobs were also failing, and the change needs to pass those tests.

**Comment by jkotas** [comment]
Closing stale PRs.

**Pull request closed without merging**